code/datums/medal_awards.dm

```diff
@@ -182,6 +182,8 @@ GLOBAL_LIST_EMPTY(jelly_awards)
 	if(give_medal_award(get_turf(printer)))
 		user.visible_message(SPAN_NOTICE("[printer] prints a medal."))
 
+GLOBAL_LIST_INIT(xeno_medals, list(XENO_SLAUGHTER_MEDAL, XENO_RESILIENCE_MEDAL, XENO_SABOTAGE_MEDAL, XENO_PROLIFERATION_MEDAL, XENO_REJUVENATION_MEDAL))
```

Comment:
If we're going to put xeno medals in a list may as well do the same for marines (see line 55)